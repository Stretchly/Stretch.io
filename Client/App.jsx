/**
 * ************************************
 *
 * @module
 * @author Eivind Del Fierro, Morah Geist
 * @date 07/2023
 * @description main app container rendering 3 child containers
 *
 * ************************************
 */

import React from 'react';
import HeaderContainer from './containers/HeaderContainer.jsx';
// import TimerContainer from './containers/TimerContainer.jsx';
import MenuContainer from './containers/MenuContainer.jsx';
import StretchContainer from './containers/StretchContainer.jsx';
import './stylesheets/application.scss';

// Init func app that returns our main containers
const App = () => {
  return (
    <div className="mainApp">
      <HeaderContainer />
      {/* <TimerContainer /> */}
      <MenuContainer />
      <StretchContainer />
    </div>
  );
};

export default App;
