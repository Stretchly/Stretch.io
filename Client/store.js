/**
 * ************************************
 *
 * @module  Store
 * @author Eivind Del Fierro, Morah Geist
 * @date 07/2023
 * @description redux store
 *
 * ************************************
 */

import { configureStore } from '@reduxjs/toolkit';
import reducers from './reducers';

const store = configureStore({ reducer: reducers });

export default store;
