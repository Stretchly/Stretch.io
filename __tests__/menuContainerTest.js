const server = 'http://localhost:3000';
const request = require('supertest');
const fs = require('fs');
import React from 'react';
import App from '../Client/App.jsx';
import { Store } from '../Client/store.js';
import { Provider } from 'react-redux';
import { render, screen, waitFor } from '@testing-library/react';
import renderer from 'react-test-renderer';


describe('test for menuContainer.jsx', () => {
  // beforeAll(() => {
  //   const renderedMenuContainer = render(<menuContainer/>);
  // })
  it('menuContainer.js renders on the page', () => {
    const tree = renderer
      .create(<menuContainer/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});