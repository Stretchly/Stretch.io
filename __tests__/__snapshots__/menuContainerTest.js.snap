// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for menuContainer.jsx menuContainer.js renders on the page 1`] = `<menuContainer />`;
